Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -124,6 +124,8 @@ public void run() {
 
         adConfig.setAdSize(vungleAdSize);
 
+        sVungleRouter.addRouterListener(mPlacementId, mVungleRouterListener);
```

Comment:
Global adding callback instance to improve loading banner/MREC if its already loaded success.